include/glow/IR/IR.h

```diff
@@ -212,6 +228,29 @@ class Instruction : public Value, public TaggedListNode<Instruction> {
 
   static bool classof(const Instruction *I) { return true; }
 
+  /// Dump a textual representation of the Instruction.
+  void dump() const;
+
+  /// Dump to string.
+  std::string toString() const;
+
+  /// Dump a textual representation of the function into provided output stream.
+  void dump(llvm::raw_ostream &OS) const;
```

Comment:
Is there a reason this parameter name is capitalized but the others aren't?

Reply:
I saw some existing dump functions in IR.h using the capitalized parameter (IRFunction). So I just wanted to keep them consistent. Should I keep this or change them all?

Reply:
> Is there a reason this parameter name is capitalized but the others aren't?

I saw some existing dump functions in IR.h using the capitalized parameter (IRFunction). So I just wanted to keep them consistent. Should I keep this or change them all?

Reply:
You used `os` everywhere else so I think you should change to `os` here.